experimenter/experimenter/test.txt

```diff
@@ -0,0 +1 @@
+test
```

Comment:
do we want to keep this file?

Reply:
❌

Reply:
It's necessary to trigger the path gates so the tests actually run, it'll be removed before merging 👍